docs/infrastructure.md

```diff
@@ -0,0 +1,82 @@
+- argo-application-controller
+
+### Helm Overview
+#### Repos
+- [Datarepo-helm](https://github.com/broadinstitute/datarepo-helm)
+  - Contains datarepo specific charts api, ui, proxy
+- [monster-helm](https://github.com/broadinstitute/monster-helm)
+  - Contains gcp managed cert chart
+- [argo-helm](https://github.com/argoproj/argo-helm/tree/master/charts/argo-cd)
+  - Contains argocd chart for continuous deployment
+- [stable/prometheus-operator](https://github.com/helm/charts/tree/master/stable/prometheus-operator)
+  - Contains helm stable which includes the prometheus-operator chart
+
+#### Definitions
+This is specifically referring to a helm `Values.yaml` file. This file contains all the user specific values to be overritten in the chart being used. Datarepo uses an umbrella chart and each users values can be found [here](https://github.com/broadinstitute/datarepo-helm-definitions)
+
+## Github Actions
+GitHub Actions automates all your workflows and jobs that happen in the back ground. Here we will cover datarepo specific actions. These are mostly for internal use such as releasing charts and bumping deployments. This doc is going to assume you understand the basics [here](https://help.github.com/en/actions).
+
+#### Datarepo Actions
+- [Jade-Datarepo integration testing](https://github.com/DataBiosphere/jade-data-repo/blob/develop/.github/workflows/gradle-build-pr.yml)
+  - On pull request
+  - This action has two blocks that will spawn two jobs one is the connected test and the other is the integration test. This will take the code from a PR and build a container then test it.
+- [Jade-Datarepo dev container build and helm definition bump](https://github.com/DataBiosphere/jade-data-repo/blob/develop/.github/workflows/dev-image-update.yaml)
+  - On Merge to develop
+  -  This action take the passed code from [Jade-Datarepo integration testing](https://github.com/DataBiosphere/jade-data-repo/blob/develop/.github/workflows/gradle-build-pr.yml) and builds a bless container specifically for develop. Once the container is build it will checkout the [datarepo-helm-definitions](https://github.com/broadinstitute/datarepo-helm-definitions) and increment the api version [tag shown here](https://github.com/broadinstitute/datarepo-helm-definitions/commit/80c0abd317981970cf979498895ab43171d8f544)
+- [Jade-Datarepo-ui integration testing](https://github.com/DataBiosphere/jade-data-repo-ui/blob/develop/.github/workflows/test-e2e.yml)
+    - On pull request
+    - This action will start a job one, the UI end to end tests. This will take the code from a PR and build a container then test it.
+- [Jade-Datarepo-ui dev container build and helm definition bump](https://github.com/DataBiosphere/jade-data-repo-ui/blob/develop/.github/workflows/dev-image-update.yaml)
+  - On Merge to develop
+  -  This action take the passed code from Jade-Datarepo-ui integration testing and builds a bless container specifically for develop. Once the container is build it
```

Comment:
What does a blessed container mean? Also maybe you didn't finish the last sentence here?

Reply:
code passed integration and is being merged to develop = blessed which in turn just means it gets a `-develop` tag appended to it.

Reply:
Understood. Could you add that to the documentation? It was a helpful explanation for me.

Reply:
added a one liner about it